exporters/tokens/exporter.json

```
@@ -1,7 +1,7 @@
 {
-  "id": "almacareer.exporter-spirit-variables-scss",
-  "name": "Spirit Varibles SCSS Exporter",
-  "description": "Spirit SCSS Exporter",
+  "id": "almacareer.spirit-tokens-exporter",
```

```suggestion
  "id": "alma.career.spirit-tokens-exporter",
```

`alma.career` or `almacareer` 🤷‍♂️ but pick for consistency in both `exporter.json`.